.github/workflows/cd-binaries.yaml

```diff
@@ -55,6 +55,7 @@ jobs:
     needs: [prepare, build]
     if: ${{ startsWith(github.ref, 'refs/tags/') }}
     steps:
+      - uses: actions/checkout@v3
```

Comment:
where is this used?

Reply:
This is used by `gh` in a subsequent step and was missed in refactoring